burn-book/src/saving-and-loading.md

```diff
@@ -0,0 +1,90 @@
+# Saving and Loading Models
+
+Saving your trained machine learning model to disk is quite easy, no matter the output format you choose. As mentioned in the [Record](./building-blocks/record.md) section, different formats are supported to serialize/deserialize models. By default, we use the `NamedMpkFileRecorder` which uses the [MessagePack](https://msgpack.org/) binary serialization format with the help of [smp_serde](https://docs.rs/rmp-serde/).
+
+```rust, ignore
+// Save model in MessagePack format with full precision
+let recorder = NamedMpkFileRecorder::<FullPrecisionSettings>::new();
+model
+  .save_file(model_path, &recorder)
+  .expect("Should be able to save the model");
+```
+
+Note that the file extension is automatically handled by the recorder depending on the one you choose. Therefore, only the file path and base name should be provided.
+
+Now that you have a trained model saved to your disk, you can easily load it in a similar fashion.
+
+```rust, ignore
+// Load model in full precision from MessagePack file
+let recorder = NamedMpkFileRecorder::<FullPrecisionSettings>::new();
+model
+  .load_file(model_path, &recorder)
+  .expect("Should be able to load the model weights from the provided file");
+```
+
+**Note:** models can be saved in different output formats, just make sure you are using the correct recorder type when loading the saved model. Type conversion between different precision settings is automatically handled, but formats are not interchangeable. A model can be loaded from one format and saved to another format, just as long as you load it back with the new recorder type afterwards.
+
+## Custom Initialization from Recorded Weights
```

Comment:
I would remove "Custom" because it applies there is an alternative normal path, which isn't the case.

Reply:
Good point! 👍